# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/test/proxy_tunnel_socket_unittest.cpp
# Commit: c684f79d8c67
# Full Hash: c684f79d8c673f21179fce00358d70845a9692c0
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Rename PWebrtcProxyChannel to PWebrtcTCPSocket. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45098
# ==============================================================================

diff -r 82af7b115f60 -r c684f79d8c67 media/mtransport/test/proxy_tunnel_socket_unittest.cpp
--- a/media/mtransport/test/proxy_tunnel_socket_unittest.cpp	Tue Sep 17 18:15:23 2019 +0000
+++ b/media/mtransport/test/proxy_tunnel_socket_unittest.cpp	Tue Sep 17 18:15:41 2019 +0000
@@ -12,7 +12,7 @@
 #include "mozilla/net/NeckoChannelParams.h"
 #include "nr_socket_proxy.h"
 #include "nr_socket_proxy_config.h"
-#include "WebrtcProxyChannelWrapper.h"
+#include "WebrtcTCPSocketWrapper.h"
 
 #define GTEST_HAS_RTTI 0
 #include "gtest/gtest.h"
@@ -45,7 +45,7 @@
     ASSERT_EQ(0, r);
 
     // fake calling AsyncOpen() due to IPC calls. must be non-null
-    mSProxy->AssignChannel_DoNotUse(new WebrtcProxyChannelWrapper(nullptr));
+    mSProxy->AssignChannel_DoNotUse(new WebrtcTCPSocketWrapper(nullptr));
   }
 
   void TearDown() override { mSProxy->close(); }